src/components/PostList/PostList.jsx

```diff
@@ -1 +1,18 @@
-export const PostList = () => <>Put the list here</>;
+import './PostList.scss';
+import { PostInfo } from '../PostInfo/PostInfo';
+import commentsFromServer from '../../api/comments.json';
+
+export const PostList = ({ posts }) => {
+  return (
+    <div className="PostList">
+      {posts.map(post => {
+        const comments = commentsFromServer.filter(
+          comment => comment.postId === post.id,
+        );
+        const postWithoutComments = { ...post, comments };
```

Comment:
move the filtering logic outside the `map`, `filter `function is called inside the `map `for every post, which can be inefficient for large datasets.